sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -4,12 +4,22 @@
 
 ### Features Added
 
+- The following methods have been added to `azure.servicebus.management.MessageCountDetails`: `deserialize`, `enable_additional_properties_sending`, `from_dict`, `is_xml_model`, `as_dict`, `serialize`.
```

Comment:
Don't document those in the ChangeLog please. While this is true, we don't want to advertised them so much.